src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -85,9 +85,10 @@ public enum CaseEvent {
     NOTIFY_APPLICANT_SOLICITOR1_FOR_CLAIM_DISMISSED(CAMUNDA),
     NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_DISMISSED(CAMUNDA),
     NOTIFY_RESPONDENT_SOLICITOR2_FOR_CLAIM_DISMISSED(CAMUNDA),
-    NOTIFY_APPLICANT_SOLICITOR1_SDO_TRIGGERED(CAMUNDA),
+    NOTIFY_APPLICANTS_SOLICITOR_SDO_TRIGGERED(CAMUNDA),
```

Comment:
Since both applicants will always have the same legal rep a second event is not needed for an applicant solicitor2